src/admin/usageStatuses/usageStatusDialog.component.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { AddUsageStatus } from '../../app.types';
+import { trimStringValues } from '../../utils';
+import handleIMS_APIError from '../../handleIMS_APIError';
+import { AxiosError } from 'axios';
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Grid,
+  TextField,
+} from '@mui/material';
+import { useAddUsageStatus } from '../../api/usageStatus';
+
+export interface UsageStatusDialogProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+function UsageStatusDialog(props: UsageStatusDialogProps) {
+  const { open, onClose } = props;
+
+  const [usageStatusDetails, setUsageStatusDetails] =
+    React.useState<AddUsageStatus>({
+      value: null,
+    });
+
+  const [valueError, setValueError] = React.useState<string | undefined>(
+    undefined
+  );
+
+  const { mutateAsync: addUsageStatus, isPending: isAddPending } =
+    useAddUsageStatus();
+
+  const handleClose = React.useCallback(() => {
+    setUsageStatusDetails({
+      value: null,
+    });
+    setValueError(undefined);
+    onClose();
+  }, [onClose]);
+
+  const handleErrors = React.useCallback((): boolean => {
+    let hasErrors = false;
+    if (
+      !usageStatusDetails.value ||
+      usageStatusDetails.value.trim().length === 0
+    ) {
+      hasErrors = true;
+      setValueError('Please enter a value');
+    }
+
+    return hasErrors;
+  }, [usageStatusDetails]);
+
+  const handleAddUnit = React.useCallback(() => {
+    const hasErrors = handleErrors();
+
+    if (hasErrors) {
+      return;
+    }
+
+    addUsageStatus(trimStringValues(usageStatusDetails))
+      .then(() => handleClose())
+      .catch((error: AxiosError) => {
+        if (error.response?.status === 409) {
+          setValueError('A usage status with the same name already exists');
+          return;
+        }
+        handleIMS_APIError(error);
+      });
+  }, [handleErrors, addUsageStatus, usageStatusDetails, handleClose]);
+
+  return (
+    <Dialog open={open} maxWidth="lg" fullWidth>
```

Comment:
Could this width be reduced, it seems a bit too wide for what it is
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/45a87659-79d2-49a3-acaa-bdf27dea4118)
perhaps md or sm look better?